cmd/server.go

```diff
@@ -36,13 +36,14 @@ goc server --port=:8080
 goc server --port=localhost:8080
 `,
 	Run: func(cmd *cobra.Command, args []string) {
-		cover.Run(port)
+		cover.Run(port, persistenceFile)
 	},
 }
 
-var port string
+var port, persistenceFile string
 
 func init() {
 	serverCmd.Flags().StringVarP(&port, "port", "", ":7777", "listen port to start a coverage host center")
+	serverCmd.Flags().StringVarP(&persistenceFile, "addr", "", "_svrs_address.txt", "path")
```

Comment:
suggest to change flag name as `local-persistence`。And make sure the file exists before writing。